src/components/Form/HorizontalFieldText/index.tsx

```diff
@@ -98,7 +98,7 @@ export class HorizontalFieldText extends React.Component<HorizontalFieldTextProp
                   data-for="horizontal-field-text-label__tooltip"
                   data-tip={tooltip}
                 />
-                <ReactTooltip effect="solid" id="horizontal-field-text-label__tooltip" />
+                <ReactTooltip effect="solid" id="horizontal-field-text-label__tooltip" html={true} />
```

Comment:
based on [react tooltip docs](https://github.com/wwayne/react-tooltip#jsx-note), i think you should wrap the tooltip with renderToString from react-dom/server